src/pages/find-id-password/FindIdForm.tsx

```diff
@@ -1,9 +1,15 @@
 import { css } from "@emotion/react";
+import { useNavigate } from "react-router-dom";
 
 import Button from "@/components/ui/Button";
 import Input from "@/components/ui/Input";
 
 export default function FindIdForm() {
+  const navigate = useNavigate();
+
+  const findIdButton = () => {
+    navigate("find-id-result");
+  };
```

Comment:
`<button>` 태그는 `<form>` 요소 안에 있을 경우, type을 명시하지 않으면 기본적으로 submit 타입으로 동작합니다.
따라서 버튼을 클릭하면 폼이 제출되며, 브라우저는 이때 페이지를 새로고침하게 됩니다.

페이지 새로고침이 아닌 리렌더링으로 화면을 교체하기 위해서 브라우저의 기본 동작을 막아주세요.

```suggestion
import { type MouseEvent } from "react";

  const findIdButton = (e: MouseEvent<HTMLButtonElement>) => {
    e.preventDefault();
    navigate("find-id-result");
  };
```